README.md

```diff
@@ -662,7 +629,6 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
    * [crystal-tools](https://atom.io/packages/crystal-tools) - Enables built in tools in Crystal compiler
    * [language-crystal-actual](https://atom.io/packages/language-crystal-actual) - Crystal language support in Atom
  * Emacs
-   * [emacs-crystal-mode](https://github.com/dotmilk/emacs-crystal-mode) - Crystal language support for Emacs
```

Comment:
It is hard to add ci build and tests for this. I would probably just whitelist it.